### DISCUSSION TITLE
GATE.IO 2 XCH black me

### DISCUSSION THREAD

**Discussion by computer817** [Chia-Network/chia-blockchain#6936] [Support]
I set the XCH mining reward address to gate.io's wallet.
15 days ago I mined 2 coins.
But the reward hasn't come yet.
Platform is the reply project block problem, has been not to deal with! Can't arrive

**Comment by loppefaaret** [comment]
do you have any addresses or anything we can use to track what is going on ?

**Reply by computer817** [reply]
0xaa99609dde2a7fcffc3b690b4e16443484f376502bdeb7746d8dc8c4feaec85d
0x72a64aa64d308cb14556727b2109fbf9a2a421d8144e4872f2e4c3ba2b682c63
This is a hash value
xch1zhlx266mld4u2z5yxpr0rym5a8u3twfgshncv9n0pzcnrs3s522swkjxu0
Gate wallet address
Shown on the chain to the account, but I don't GATE purse
And gate explained program block problem, need time to solve! Still have 15 days
In the community saw many similar experiences of friends, I want to gate may really have a problem! Not alone

**Reply by loppefaaret** [reply]
https://xchscan.com/address/xch1zhlx266mld4u2z5yxpr0rym5a8u3twfgshncv9n0pzcnrs3s522swkjxu0
as you can see here, there is no "from" on the rewards from a block win - they are special in that they aren't really transactions, they are rewards. and are handled differently - it is solely up to gate.io to choose what to do here.
But it is generally discouraged to farm directly into an exchange wallet, exactly for these reasons!
get your farming rewards paid out to a wallet you control!
the only one that can possibly help you here is gate.io - but technically, you did do a "reward" to the address, and not a "transaction"

**Reply by computer817** [reply]
Gate. The problem of the IO back block, there has been no deal with our problems, chia project side gate and whether to have the official communication channels, can help us to solve this problem, this problem still more

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
